apkid/rules/dex/obfuscators.yara

```diff
@@ -241,3 +241,20 @@ rule aamo_str_enc : obfuscator
   condition:
     1 of ($opcodes*) and all of ($a, $b)
 }
+
+rule md5obfuscator : obfuscator
+{
+    meta:
+      description = "MD5 obfuscator"
+      example     = "843a6562b62932df8b4c787466208a0523c1d88401f8cbf86f36de84ed4b7ccd"
+      author      = "Eduardo Novella"
+
+    strings:
+      // md513d0258903c37fed2a3d17a14e8551a2
+      $package = /Lmd5[a-f0-9]{32}/
+
+    condition:
+        all of them and is_dex
```

Comment:
This will detect any dex that has 'md5' with a valid md5 afterwards, which seems likely to FP. Is there anything else that could be added to this rule like parts of the decoder, using more of the class path, null bytes, etc.?

Reply:
This answers about 1% of my question, but even from this it's possible to suggest that the rule could require many instances of these strings, and since all the strings are the same length, you could even make them full dex strings while hardcoding the leb128 length.

Reply:
Dont know how to count number of instances, but it would be great to do so. Seriously! Do you know how to do that?

I'm going to modify the regex, no worries.

Reply:
Done!